lib/avatarDialog/avatar_dialog.dart

```diff
@@ -3,19 +3,18 @@ import 'package:flutter/material.dart';
 import 'package:scroll_to_index/scroll_to_index.dart';
 
 class AvatarDialog extends StatefulWidget {
-  const AvatarDialog({Key? key, required this.serverIndex}) : super(key: key);
-
-  final int serverIndex;
+  const AvatarDialog({Key? key}) : super(key: key);
 
   @override
   State<AvatarDialog> createState() => _AvatarDialogState();
 }
 
 class _AvatarDialogState extends State<AvatarDialog> {
   final AutoScrollController controller = AutoScrollController();
-  int listLength = 30;
+  int listLength = 20;
 
   int selectedIndex = -1;
+  int lastSelectedIndex = 19;
 
```

Comment:
what is selectedIndex and lastSelectedIndex?
Are they both necessary

Reply:
removed lastSelectedIndex

selectedIndex is for onTap selected index